src/data_tag.c

```diff
@@ -63,7 +63,7 @@ static void gpsd_client_event(struct mg_connection *nc, int ev, void *ev_data)
             // Success
             fprintf(stderr, "GPSd Connected...\n");
             if (ctx->init_str && *ctx->init_str) {
-                mg_send(nc, ctx->init_str, strlen(ctx->init_str));
+                mg_send(nc, ctx->init_str, (int)strlen(ctx->init_str));
```

Comment:
This one should not be required if mongoose was consistent in its methods declarations: `mg_send_http_chunk`, `mg_send_websocket_frame` and almost all others take a `size_t`, but for some reason `mg_send` does not...